css-scroll-snap-2/Overview.bs

```diff
@@ -193,26 +193,117 @@ The 'scroll-start' property {#scroll-start}
 The 'scroll-start-target' property {#scroll-start-target}
 -------------------------------------------
 
-  <pre class="propdef shorthand">
-  Name: scroll-start-target
-  Value: [ none | auto ]{1,2}
-  </pre>
-
-  This property is a shorthand property that sets all of the scroll-start-target-* longhands in one declaration.
-  The first value defines the scroll starting point in the block axis,
-  the second sets it in the inline axis.
-  If the second value is omitted, it defaults to ''none''.
-
-  Values are defined as follows:
-
-  <dl dfn-type=value dfn-for="scroll-start-target, scroll-start-target-x, scroll-start-target-y, scroll-start-target-block, scroll-start-target-inline">
-    <dt><dfn>none</dfn>
-    <dd>Element is not a ''scroll-start-target''.
-    <dt><dfn>auto</dfn>
-    <dd>Element is used to calculate the ''scroll-start'' position, 
-    taking into account ''scroll-padding'' or ''scroll-margin'' ,
-    same as a ''scroll-snap'' target.
-  </dl>
+	<h4 id="initial-scroll-target">Initial scroll target</h4>
+
+		The <dfn>initial scroll target</dfn> of a <a>scroll container</a> |scrollcontainer|
+		with respect to a particular axis is an element or pseudo-element whose {{scroll-start-target}}
```

Comment:
i've been asking the community and hunting for examples, here's the current thoughts:
1. many examples of wanting to set the block/inline scroll start targets can be achieved by specifying a single intersecting element. but, knowing which element that is this at this intersection might not be known.
2. consider the example where a scroll-start-target is desired on x but not y. so even tho it's a scroller in both axes, it'd be desirable to only scroll-start one axis at a provided element. like a table and wanting to scroll-start at a row and not having it move the x scroll position.

Reply:
Thanks for looking around on this Adam. I was wondering, if an author wants to scroll to whatever element is at a particular position, then I guess, in a sense, it's not so much the element they want to scroll to as it is the position. In this case, perhaps an acceptable pattern would be to put a placeholder element at that location and have it be the scroll-start-target. I guess there are multiple ways of constructing a grid and I'm not yet sure if this would be a feasible/acceptable approach to all(any?) of them.